### ISSUE TITLE
Volume management bug: manipulating volume switches Repeat state

### ISSUE THREAD

**Issue by cramermarius** [Rigellute/spotify-tui#121]
Just encountered this little non-urgent bug: Manipulating the volume controls through + or - switches 'Repeat: Track' to 'Repeat: All'. No change was observed when manipulating volume when in neither 'Repeat: Off' nor 'Repeat: All', in both cases the state is constant.

**Comment by sputnick1124** [comment]
What device are you using to play it when this happens. I just noticed a similar behavior using `spotifyd` as my device, but playing audio through the  web player works fine.

Actually, using the web player as the client and `spotifyd` as the player, I am unable to even select `Repeat: Track`. It just cycles between `All` and `None`. Curious...

**Comment by cramermarius** [comment]
I'm not quite sure what you mean by device in this regard. In any case, I am using a XMG a507 running popOS 19.04, using spotifyd as the server and spotify-tui as the CLI.

**Comment by sputnick1124** [comment]
Sorry for the lack of clarity. By device I meant what Spotify calls a device when it says "Devices available". In this case, the `spotifyd` server using the Spotify Connect API would be the device.

I suspected that may be the case. The issue actually lies within spotifyd (actually librespot), but has been addressed with a recent update (see https://github.com/Spotifyd/spotifyd/issues/381). I have tested this on my local machine by updating spotifyd to current master and building, but the issue has not resolved itself.

That said, I don't think there is any bug within spotify-tui to address here.
